src/command/services/command-create-list-item.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Configuration } from '../../configuration';
+import { CommandGenerateService } from './command-generate.service';
+import { ValidatorPresentService } from '../../validator/validator.module';
+import { ListItemPayload } from '../../payload/models/list_item_payload';
+
+export const CommandCreateListItemService = {
+  call: (
+    controller,
+    options: ListItemPayload,
+    configuration: Configuration
+  ) => {
+    ValidatorPresentService.call(options, [
+      'list_id',
+      'primary_value',
```

Comment:
could we validate the `list_id` path param?  We want to keep other validation on the API until we prepare some more robust validation functionality.

Reply:
Right now I'm only asserting it is defined. What other validations you suggest I do?